server/middlewares/articles/findArticle.js

```diff
@@ -6,21 +6,20 @@ import { serverError, httpResponse } from "@helpers/http";
 
 const { Articles } = models;
 export const findArticle = async (req, res, next) => {
+  const slug = req.params.slug || req.body.slug;
   try {
     const article = await Articles.findOne({
-      where: {
-        slug: req.params.slug,
-        isPublished: true
-      }
+      where: { slug }
```

Comment:
you have removed isPublished true, we need it.

Reply:
As this is the only middle that allows fetching article without restriction, I think it is wise to let the programmer decide what to do if the article is not published? A problem we may run into in the future is that if for any reason we want to delete an unpublished article, then we will never be able to find it since this middle only fetches published article

Reply:
Maybe this middlewares name can be renamed to **findPublished** article instead of **findArticle**

Reply:
you still have not implemented the feedback since morning. and you are aware, we are running out of time.